### PR TITLE
Document common properties

### PR DESCRIPTION
**Pull request by mxm**

### CODE REVIEW

**Comment by nslater** [comment]
@mxm can you rebase your changes off of my work in crate/crate-jdbc/pull/261 please?

**Comment by mxm** [comment]
Ah seems like you already did similar work, will rebase then.

**Comment by nslater** [comment]
looks like this is the better approach after all

just commenting so the ticket reflects what was discussed today

**Comment by mxm** [comment]
@nslater Could you review this then, so we can merge it?

File: docs/connecting.txt [hunk 1]
```diff
@@ -130,4 +130,16 @@ CrateDB JDBC driver supports following properties:
     only supported transaction isolation level is ``TRANSACTION_NONE``.
     Read-only connections aren't supported either.
 
+:user:
+  **Default**: The user logged into the OS
```
Comment (nslater) [inline docs/connecting.txt]: "The" -> "the"

File: docs/connecting.txt [hunk 2]
```diff
@@ -130,4 +130,16 @@ CrateDB JDBC driver supports following properties:
...
+  Specifies the user to identify against CrateDB. You may want to set this to ``crate`
```
Comment (nslater) [inline docs/connecting.txt]: "Specifies the CrateDB user."

Comment (nslater) [reply docs/connecting.txt]: "to \`\`crate\`\`,"

(notice extra tick, also comma)

File: docs/connecting.txt [hunk 3]
```diff
@@ -130,4 +130,16 @@ CrateDB JDBC driver supports following properties:
...
+  which is the default user if none have been created yet.
+
+:password:
+  Sets the password to authenticate against the CrateDB cluster.
```
Comment (nslater) [inline docs/connecting.txt]: "The password"

**Comment by mxm** [comment]
@nslater Thanks. Should be good to go now.

**Comment by mxm** [comment]
@nslater fixed

**Pull request merged**